src/mission.cpp

```diff
@@ -249,6 +252,45 @@ void mission::wrap_up()
     u.on_mission_finished( *this );
     std::vector<item_comp> comps;
     switch( type->goal ) {
+        case MGOAL_FIND_ITEM_GROUP: {
+            inventory tmp_inv = u.crafting_inventory();
+            std::vector<item *> items = std::vector<item *>();
+            tmp_inv.dump( items );
+            Group_tag grp_type = type->group_id;
+            itype_id container = type->container_id;
+            bool specific_container_required = container.compare( "null" ) != 0;
```

Comment:
Have you tried `container != "null"`?

Reply:
just tried it, it works. implemented in latest commit.